csss-site/src/about/README.md

```diff
@@ -0,0 +1,266 @@
+
+## 1.2. How Director of Archives generates `UnProcessedOfficer`
+<img src="https://docs.google.com/drawings/d/e/2PACX-1vTYsdFf2P5EiQrNwqNOYNGXOc0-E9fgUgw2mOIfSNc8WE3kSBuYuwew3w60W5HO91-WfrfqpowlZZkC/pub?w=564&amp;h=331">  
+
+The way I coded the site, each new term, all officers have to enter their information, regardless of whether they are a new officer or are just continuing to serve a term until its fullest [for example, any of the Director positions in the Fall or Spring term]
+
+Shortly before the new term, the Director of Archives had to create a new `UnProcessedOfficer` object, that they would create those entries via https://sfucsss.org/about/specify_new_officers
+
+Once the Director of Archives fills out the below form, in order to motivate/ensure repeat officers do fill in their info again for the new term, the website uses the [`UnProcessedOfficer` model in order to keep ensure that officers who have not filled in their form cannot access the SFU CSSS Google Drive, the SFU CSSS Github Org, and are not given their discord exec role](../csss/views/privilege_validation/list_of_officer_details_from_past_specified_terms.py)
+
+![](https://github.com/CSSS/csss-site/blob/add_docu/csss-site/src/about/documentation_images/generate_unprocessedofficer.png)
+
+### 1.2.1. Explanations of input fields
+Most fields in the above screenshot are pretty intuitive, however
+ * **Number of Nags so Far**: what I found was that officers tend to procrastinate with filling in their links [at least prior to tying management of the discord officer roles to whether or not they had filled in their links]. As a result, I created a [cron service](management/commands/nag_officers_to_enter_info.py) that DMs an officer using a custom cron trigger until they fill in their links.
+ * **Start Date**: So the way I coded the website was that the start_date of an officer is the date that they got elected or appointed to the position. So if a Director stuck with a position through to the end of their term, their started date for all three of their `Officer` objects for that particular run all share the same start date but have different term object. Hence why the `Officer did not have to be voted into position again this term` checkbox exists, so that the Director of Archives can just click that instead of having to interact with the date field at the beginning of every Fall and Spring term for all the Director positions that need have links created
+ * **Overwrite Current Officer**: I disabled that, it was stupid logic but just never got around to removing it from the front-end.
+
+### 1.2.2. Validations Performed
+The website ensures that the `SFU Computing ID` inputted by the Director of Archives is valid via API `GET https://rest.its.sfu.ca/cgi-bin/WebObjects/AOBRestServer.woa/rest/amaint/namespace.json?id=<sfu_computing_id>&art=<SFU_API_TOKEN>`
```

Comment:
as far as I am aware, there is no API documentation for this endpoint. Or if there is, I was never given access to it.

I learnt about it via my email communications with the FAS IT people [which I conducted via `csss@sfu.ca` and are in the `Sys-Admin Emails` folder of that inbox].

I wil make a private google doc indicating what I learnt about the SFU endpoints since I assume IT people wouldn't want that info public.